src/Symfony/Component/Workflow/DefinitionBuilder.php

```diff
@@ -61,11 +61,9 @@ public function clear()
     }
 
     /**
-     * @param string|string[]|null $initialPlaces
```

Comment:
The annotation is still more precise than the union type: It tells me that the array is expected to be an array of strings. I would keep it, just as you've done on `Definition::__construct()`.

Reply:
reverted